server/services/zigbee2mqtt/utils/convertValue.js

```diff
@@ -14,11 +14,32 @@ function convertValue(feature, state) {
       result = state === 'ON' ? 1 : 0;
       break;
     }
-    default:
+    // case 'water_leak': {
+    //   result = state === true ? 1 : 0;
+    //   break;
+    // }
+    // case 'contact': {
+    //   result = contact === true ? 1 : 0;
+    //   break;
+    // }
+
+    default: {
       if (typeof state === 'string') {
         throw new Error(`Zigbee2mqqt don't handle value "${state}" for feature "${feature}".`);
       }
-      result = state;
+      // On généralise le cas où l'état vaut true ou false => 1 ou 0
+      if (typeof state === 'boolean') {
+        if ( state === true ) { 
+          result = 1; 
+        } else if (state === false) {
+          result = 0;
+        }
+      }
```

Comment:
Can be one lined : `result = state ? 1 : 0;`

Reply:
For my 1st contrib, I wanted to be clear ! ;-)